web/srv/api_handlers.go

```diff
@@ -21,6 +21,8 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
+// Control Frame payload size can be no bigger than 125 bytes. 2 bytes are
+// reserverd for the status code when formatting the message.
```

Comment:
i think the 2 bytes are for payload size:
https://github.com/gorilla/websocket/blob/6a67f44b69004415802d2e678e483065b8fff6c8/conn.go#L1159-L1162

```
// Control Frame payload size can be no bigger than 125 bytes. 2 bytes are
// reserved for the payload size when formatting the message.
```

Reply:
@siggy `FormatCloseMessage` takes the status code we are closing the socket with and the message. It copies the close code into the first two bytes of the buffer, and then the message into the rest. That message is what Gorilla validates to be no longer than 125 bytes.

Maybe I'm missing something, but it doesn't seem like is related to the size of the payload?

Reply:
Ah you're right, I misread `FormatCloseMessage`.